setup.py

```diff
@@ -5,9 +5,14 @@
 # fmt: off
 REQUIREMENTS = [
     "attrs",
-    "click>=7,<8",
+    "click",
```

Comment:
please don't remove the version requirement.

you might want to bump the `8` to `9`, if you know that click 9.x doesn't break anything

Reply:
+1 for setting `"click>=7,<9"`